opa/tests/test.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+
+TESTDIR="$(dirname "${0}")"
+PLANDIR="$(dirname "${TESTDIR}")"
+SKIPBUILD=${SKIPBUILD:-0}
+
+source "${TESTDIR}/helpers.bash"
+
+hab pkg install --binlink core/bats
+
+hab pkg install core/busybox-static
+hab pkg binlink core/busybox-static nc
+hab pkg install -b core/jq-static core/curl
+
+# Wait for supervisor to start
+echo "Waiting for supervisor to start"
+wait_listen tcp 9632 30
+
+source "${PLANDIR}/plan.sh"
+# Unload the service if its already loaded.
+hab svc unload "${HAB_ORIGIN}/${pkg_name}"
+
+if [ "${SKIPBUILD}" -eq 0 ]; then
+  set -e
+  pushd "${PLANDIR}" > /dev/null
+  build
+  popd > /dev/null
+  set +e
+fi
+
+source results/last_build.env
```

Comment:
This is actually (potentially) loading the wrong `last_build.env` as its not run in the same directory as the `build` command.

Looking at this, we might have to change some other tests as well to fix this problem. Since the build is run in `${PLANDIR}` you might need to source it from that path, or move this line to immediately under the `build` command.

You can test this by running the build from the root of core plans, as well as inside the plan directory:

```
cd /path/to/core-plans
hab studio enter

# Inside studio: Test 1
./opa/tests/test.sh

# Inside studio: Test 2
cd opa
./tests/test.sh
```

Other than that, this looks good 👍

Reply:
Thanks! I'll fix the others once this is done :)

Reply:
@predominant if you could have another look, I'd be grateful 🎉